tinyinst.cpp

```diff
@@ -850,6 +850,8 @@ void TinyInst::OnModuleInstrumented(ModuleInfo* module) {
       }
       if(address) {
         resolved_hooks[address] = hook;
+      } else {
+        WARN("Could not resolve function %s in module %s", hook->GetFunctionName().c_str(), hook->GetModuleName().c_str());

```

Comment:
I think it only makes sense to issue this warning if hook->GetModuleName() != "*". Could you add this check?